feg/gateway/services/session_proxy/servicers/multiple_session_controller.go

```diff
@@ -219,12 +220,14 @@ func (srv *CentralSessionControllers) Enable(
 	ctx context.Context,
 	void *orcprotos.Void,
 ) (*orcprotos.Void, error) {
-	multiError := merrors.NewMulti()
+	multiError := &multierror.Error{}
 	for i, controller := range srv.centralControllers {
 		_, err := controller.Enable(ctx, void)
-		multiError = multiError.AddFmt(err, "error(%d):", i+1)
+		if err != nil {
+			multiError = multierror.Append(multiError, errors.Errorf("error(%d): %v", i+1, err))
```

Comment:
`multiError = multierror.Append(multiError, ...` is weird to read, with `multi{eE}rror` in different casing variants. Is there a neater way to write that or a more distinct name for the variable?

Reply:
Fixed.